frontend/public/components/monitoring/metrics.tsx

```diff
@@ -992,14 +995,17 @@ const QueriesList = connect(({ UI }: RootState) => ({
   count: UI.getIn(['queryBrowser', 'queries']).size,
 }))(QueriesList_);
 
-const PollIntervalDropdown = connect(
-  ({ UI }: RootState) => ({
-    interval: UI.getIn(['queryBrowser', 'pollInterval']),
-  }),
-  {
-    setInterval: UIActions.queryBrowserSetPollInterval,
-  },
-)(IntervalDropdown);
+const PollIntervalDropdown = () => {
```

Comment:
This one component was switched to `useDispatch` and `useSelector` to resolve a type error. Other uses of `connect` in this file are being left for a future PR.